### PR TITLE
Update the tutorials

### PR DESCRIPTION
**Pull request by alihamdan**
A rewrite of the tutorials.
The Getting Started page is now more organized with a summary at the beginning and some useful notes through out. The other pages are made up to date with the code.

@benoit9126 I didn't (couldn't) run the notebooks locally, could you please test them on your computer?

### CODE REVIEW

**Review by benoit9126 — APPROVED** [review]

**Comment by benoit9126** [comment]
@alihamdan Could you please update the README of the repository? 
If you have any comments/improvements on the issue templates, do not hesitate to correct them.

**Comment by alihamdan** [comment]
> @alihamdan Could you please update the README of the repository?

The README on develop is already up-to-date. Do you have anything in mind to add?

**Comment by alihamdan** [comment]
I did some minor tweaks to the tutorials and the issue templates and added a diagram for the network the Advanced section

File: roseau/load_flow/network.py [hunk 1]
```diff
@@ -46,104 +46,99 @@
...
+                    "iterations": 2,
+                    "target_error": 1e-06,
+                    "final_error": 1.9468870959826745e-12
+                }
     """
 
     DEFAULT_PRECISION: float = 1e-6
     DEFAULT_MAX_ITERATIONS: int = 20
-    DEFAULT_BASE_URL: str = "https://load-flow-api.roseautechnologies.com/"
+    DEFAULT_BASE_URL: str = "https://load-flow-api-dev.roseautechnologies.com/"
```
Comment (alihamdan) [inline roseau/load_flow/network.py:141]: @benoit9126 was this change supposed to be commited?

**Review by alihamdan — COMMENTED** [review]
Looks great, thanks.

Let’s merge this PR. I’ll do another quick pass on the docs tomorrow. We can do a final retouch in a new PR.

**Pull request merged**